pgx-examples/aggregate/.gitignore

```diff
@@ -0,0 +1,7 @@
+.DS_Store
+.idea/
```

Comment:
```suggestion
.idea/
.vscode/
```

Reply:
This is from our standard template, if you'd like to change that, please open an issue.